documentation/asciidoc/topics/yaml_annotated/cr_backup_site.yaml

```diff
@@ -8,9 +8,9 @@ spec:
         expose:
           type: LoadBalancer <3>
       locations: <4>
-      - name: SiteA <5>
```

Comment:
You forgot to update `.sites.local.name` above?

Reply:
great catch. fixed.